tests/FakeItEasy.Specs/SelfInitializedFakesSpecs.cs

```diff
@@ -34,10 +34,10 @@ public static class SelfInitializedFakesSpecs
                         realServiceWhileRecording = A.Fake<ILibraryService>();
                         realServiceDuringPlayback = A.Fake<ILibraryService>();
 
-                        A.CallTo(() => realServiceWhileRecording.GetCount("9780345813923"))
-                            .ReturnsNextFromSequence(11, 10);
-                        A.CallTo(() => realServiceWhileRecording.GetCount("9781593078225"))
-                            .Returns(3);
+                        A.CallTo(() => realServiceWhileRecording.GetCount("1"))
+                            .ReturnsNextFromSequence(0x1A, 0x1B);

```

Comment:
Ah, I see. No, it's fine as is. The connection between the input and output just didn't leap out at me.